tests/Parser.test.cpp

```diff
@@ -905,6 +905,87 @@ TEST_CASE_FIXTURE(Fixture, "parse_compound_assignment_error_multiple")
     }
 }
 
+TEST_CASE_FIXTURE(Fixture, "parse_interpolated_string_double_brace_begin")
+{
+    ScopedFastFlag sff{"LuauInterpolatedStringBaseSupport", true};
+
+    try
+    {
+        parse(R"(
+            _ = `{{oops}}`
```

Comment:
Speaking of, as a sanity check, you might want a test for `` `{{oops}` ``. This looks _weird_ to look at, but we want to make sure it is still properly closing two braces.